data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -36,14 +36,14 @@ spec:
             - -c
             - |
               echo 'Installing bridge and tuning CNIs'
-              cp -f /usr/src/containernetworking/plugins/bin/*bridge /opt/cni/bin/ || exit 1
-              cp -f /usr/src/containernetworking/plugins/bin/*tuning /opt/cni/bin/ || exit 1
+              cp -f /usr/src/containernetworking/plugins/bin/bridge {{ .CNIBinDir }}/cnv-bridge || exit 1
+              cp -f /usr/src/containernetworking/plugins/bin/tuning {{ .CNIBinDir }}/cnv-tuning || exit 1
               # Some projects (e.g. openshift/console) use cnv- prefix to distinguish between
               # binaries shipped by OpenShift and those shipped by KubeVirt (D/S matters).
               # Following two lines make sure we will provide both names when needed.
-              find /opt/cni/bin/cnv-bridge || ln -s /opt/cni/bin/bridge /opt/cni/bin/cnv-bridge
-              find /opt/cni/bin/cnv-tuning || ln -s /opt/cni/bin/tuning /opt/cni/bin/cnv-tuning
-              echo "Entering sleep... (success)"
+              find {{ .CNIBinDir }}/bridge &>/dev/null || ln -s {{ .CNIBinDir }}/cnv-bridge {{ .CNIBinDir }}/bridge
```

Comment:
Using `find` instead of testing for a file to exist is really odd. But as it was used before, I guess it is not a must to change it now.

Reply:
yeah, we can always make improvements in another PR